### PR TITLE
fix 88 gender name

### LINKED ISSUE #88: Fix the gender name in the member model
The name are presently in french they need to be in english.

### PR DESCRIPTION
**Pull request by maximest-pierre**
fixed #88 and also change the default value to unknown.

### CODE REVIEW

File: django/santropolFeast/member/models.py [hunk 1]
```diff
@@ -36,11 +36,11 @@ class Meta:
...
         choices=(
-            ('H', 'Homme'),
-            ('F', 'Femme'),
-            ('U', 'Inconnu'),
+            ('M', 'Male'),
+            ('F', 'Female'),
+            ('U', 'Unknown'),
         ),
-        default=1
+        default=2
```
Comment (RignonNoel) [inline django/santropolFeast/member/models.py]: I'm not sure on this default value:

 - When i create a member in the python shell, the member.gender value is u'2'
 - When i create a member in the admin panel, the default value in the dropdown is 'Male'

File: django/santropolFeast/member/models.py [hunk 2]
```diff
@@ -36,11 +36,11 @@ class Meta:
     gender = models.CharField(
         max_length=1,
         choices=(
-            ('H', 'Homme'),
-            ('F', 'Femme'),
-            ('U', 'Inconnu'),
+            ('M', 'Male'),
```
Comment (RignonNoel) [inline django/santropolFeast/member/models.py]: The 'M' is ok but for the rest we need to have identifiants and not the english version!

So you need to :
 - Delete the caps of the word
 - Add the translation function on each ID

File: django/santropolFeast/member/models.py (hunk 1 above)
Comment (maximest-pierre) [inline django/santropolFeast/member/models.py]: The django doc said that we must call a function that returns the default value.

ref. https://docs.djangoproject.com/en/1.9/ref/models/fields/#django.db.models.CharField

Under the section default.

Comment (RignonNoel) [inline django/santropolFeast/member/models.py]: I've find this example in the same page :
```
class Student(models.Model):
    FRESHMAN = 'FR'
    SOPHOMORE = 'SO'
    JUNIOR = 'JR'
    SENIOR = 'SR'
    YEAR_IN_SCHOOL_CHOICES = (
        (FRESHMAN, 'Freshman'),
        (SOPHOMORE, 'Sophomore'),
        (JUNIOR, 'Junior'),
        (SENIOR, 'Senior'),
    )
    year_in_school = models.CharField(max_length=2,
                                      choices=YEAR_IN_SCHOOL_CHOICES,
                                      default=FRESHMAN)
```

Comment (maximest-pierre) [inline django/santropolFeast/member/models.py]: I will implement it like the example.

Comment (manumilou) [inline django/santropolFeast/member/models.py]: @maximest-pierre  Check out the Meal Model, the same thing has already been implemented: https://github.com/savoirfairelinux/santropol-feast/blob/dev/django/santropolFeast/meal/models.py#L23

**Pull request merged**